all/entrypoint.sh

```diff
@@ -143,6 +143,10 @@ case $GEN_LANG in
     "java")
         GEN_STRING="--grpc_out=$OUT_DIR --${GEN_LANG}_out=$OUT_DIR --plugin=protoc-gen-grpc=`which protoc-gen-grpc-java`"
         ;;
+    "node")
+        SERVICE_NAME=$(basename $PROTO_DIR)
```

Comment:
I don't think you need this `SERVICE_NAME` line.

Reply:
indeed - it's a leftover. I'll remove.